app/models/shelter.rb

```diff
@@ -1,4 +1,4 @@
 class Shelter < ApplicationRecord
   validates_presence_of :name
-  has_many :pets
+  has_many :pets, dependent: :delete_all
```

Comment:
Note to self: this will now delete any pets that belong to a shelter if that shelter is deleted from the database